sample/src/main/java/com/github/paylike/sample/MainActivity.kt

```diff
@@ -0,0 +1,23 @@
+package com.github.paylike.sample
+
+import android.os.Bundle
+import android.util.Log
+import androidx.appcompat.app.AppCompatActivity
+import com.github.paylike.kotlin_currencies.PaylikeCurrencies
+import com.github.paylike.kotlin_currencies.PaylikeCurrency
+import com.github.paylike.kotlin_currencies.generated.CurrencyCode
+import com.github.paylike.kotlin_money.Money
+import com.github.paylike.kotlin_money.PaymentAmount
+
+class MainActivity : AppCompatActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+
+        val eur: PaylikeCurrency = PaylikeCurrencies.byCode(CurrencyCode.EUR)
```

Comment:
I am not convinced that here it make sense to create a new instance for the bundle. Feels like we are trying to render something unnecessarily.

Reply:
You mean the "onCreate(savedInstanceState: Bundle?)"? This activity is generated automatically by android studio but i take a look at it to if its necessary

Reply:
I don't mean the sample overall, I mean that this looks like to me that we are trying to run a sample android application (a test instance) that utilises the library. My question is, do we need to run that instance? As I see it would be enough to just print the value out as there is nothing to render. However, if this is the best practice then I am fine with it.

Reply:
So far i haven't seen other practices to show a sample and run it on an android device.